.github/workflows/build.yml

```diff
@@ -18,14 +18,12 @@ jobs:
     strategy:
       matrix:
         image:
-          - "ghcr.io/ivarconr/unleash-enterprise:5.9"
           - "unleashorg/unleash-server:5.9"
-          - "ghcr.io/ivarconr/unleash-enterprise:5.12"
           - "unleashorg/unleash-server:5.12"
-          - "ghcr.io/ivarconr/unleash-enterprise:6.0"
           - "unleashorg/unleash-server:6.0"
-          - "ghcr.io/ivarconr/unleash-enterprise:latest"
           - "unleashorg/unleash-server:latest"
+          - "ghcr.io/ivarconr/unleash-enterprise:6.8"
```

Comment:
I've kept server support back to 5.9 but new API behaviour is needed from 6.8 :(

Reply:
It'd be nice to have tests that run only when tested from a min version onward, similar to what we do for enterprise only. I think that also serves as valid documentation.

:thinking: 

Or maybe it has to be the other way around, the terraform resource will only do something if the min version matches. Now I think it'll fail.

Reply:
So something like this?

isEnterprise() && versionAbove(6.8)
// some test code